mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -214,6 +214,15 @@ def run_task(self):  # {{{
             x = self.lat[region]
             y = self.depth
             z = self.moc[region]
+            # Subset lat range
+            minLat = config.getExpression(self.sectionName,
+                                         'latBinMin{}'.format(region))
+            maxLat = config.getExpression(self.sectionName,
+                                         'latBinMax{}'.format(region))
+            indLat = np.logical_and(x >= minLat, x <= maxLat)
+            x = x[indLat]
+            z = z[:, indLat]
```

Comment:
I do not have xarray datasets at this point, so I am subsampling x and z individually.